akka-actor-typed/src/main/scala/akka/actor/typed/internal/ActorContextImpl.scala

```diff
@@ -32,6 +32,7 @@ import org.slf4j.LoggerFactory
   // lazily initialized
   private var logger: OptionVal[Logger] = OptionVal.None
   private var mdcUsed: Boolean = false
+  private var tags: OptionVal[String] = OptionVal.None
```

Comment:
maybe we can save a field by putting this into the `logger` field, making that a `OptionVal[(Logger, String)]` or a more specific class instead of tuple. Only saves memory if logging isn't used, but I also think they belong together and could be initialized together.

`mdcUsed` could also be moved into that class, but should be a mutable field to avoid allocations

Reply:
I agree, that makes sense.